### ISSUE TITLE
O button pauses emulator

### ISSUE THREAD

**Issue by Surnis** [unofficial-issue-tracker/duckstation#185]
All used to work well with duckstation, but suddenly, the O button decided it was used to pause the emulator, same as pressing pause in the ribbon on top. I verified controller bindings, redone them, tried changing between SDL, Xinput or Dinput, tried between anolog and digital controller, updated and even even reinstalled the emulator, and nothing works. 
I am using a real PS4 controller. 
I seem to remember having this issue a long time ago, but I can't remember what was done to get out of it... Any help would be appreciated. Thanks!

**Comment by JukePlz** [comment]
Most likely there's some third party tool interfering and sending a keystroke.

First check the Hotkeys tab in the controller configuration, and see if your "Toggle Pause" function is bound to the default "Keyboard/Space" or something else. If it's on the default spacebar key then I'd wager you have something like Steam Input trying to use your DuckShock4 as a keyboard, so it sends both the configured key for O as well as the Space key when you press it.

**Comment by Surnis** [comment]
Indeed, the hotkey was spacebar. I'm not sure why Steam would suddenly mess with my controller like this, or what I can do to stop it, but by changing the hotkey to something else (P in this case, something that feels natural if I do want to pause.) it seems to have solved the problem. Would prefer to not bind a hotkey at all, but that doen't seem possible.
That would mean that each time I press O, the computer still presses spacebar also, but at least it does nothing! 
This third party interference must be true, because I noticed since a similar problem with Ralibretro, where the menu opens when I press start on the controller. I really must find out how to turn that thing off! But that's a problem for another forum, I guess! 
At least, it's solved for Duckstation! Thanks a lot for your help!
